### ISSUE TITLE
Change Early Net Zero description to Net Zero Early

### ISSUE THREAD

**Issue by mary-black** [odileeds/northern-powergrid#51]
I changed it in the scenarios index file but got an error saying it was undefined in index.json - though I could still use it in the drop down list. Can we change the name - no need to change file names, just the descriptor people see on screen.

Where else should I have changed it?

**Comment by mary-black** [comment]
Do I just change the name in config.js too?

**Comment by slowe** [comment]
The scenario names in `index.json` are unique keys that we are using elsewhere. To make things simpler previously we used the same key to be the scenario title.

The visualisation defaults (scenario, view, parameter, start year, year range, map bounds etc) and specific customisation are all set in `resources/config.js`. I have had to restructure things compared to DFES 2019 in order to generalise the code and make it more reusable for e.g. 2020 Emerging Thinking. So things are now split in various places.
